### ISSUE TITLE
Styles refactroing

### ISSUE THREAD

**Issue by marharyta** [CSCfi/rems#2588]

**Comment by marharyta** [comment]
Oh, I also believe we did discuss at one point to get rid of the word ```generate``` in the style defs since it doesn't add any value, correct me if I am wrong @opqdonut ?

**Comment by opqdonut** [comment]
Yeah. Not a big enough refactor to do on its own IMO, but if somebody is touching the style code it can be done at the same time.

**Comment by Macroz** [comment]
Let's see if there is anything to be done from here if we do UI refresh.
